vyper/ir/compile_ir.py

```diff
@@ -1048,4 +1051,7 @@ def assembly_to_evm(assembly, start_pos=0):
     assert len(o) == pos - start_pos, (len(o), pos, start_pos)
     line_number_map["breakpoints"] = list(line_number_map["breakpoints"])
     line_number_map["pc_breakpoints"] = list(line_number_map["pc_breakpoints"])
+    if insert_vyper_signature:
+        # CBOR encoded: {"VYPR": [major,minor,patch]}
+        o += b"\xa1\x64VYPR\x83" + bytes(list(version_tuple))
```

Comment:
The name of the compiler is `vyper`
```suggestion
        # CBOR encoded: {"vyper": [major,minor,patch]}
        o += b"\xa1\x64vyper\x83" + bytes(list(version_tuple))
```

Reply:
ah, then it needs to be `\x65` (aka bytes(5))

Reply:
```suggestion
        # CBOR encoded: {"vyper": [major,minor,patch]}
        o += b"\xa1\x65vyper\x83" + bytes(list(version_tuple))
```